release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -328,11 +328,23 @@ In the table, features are marked with the following statuses:
 |DEP
 |DEP
 
+|v1beta1 CRDs
+|GA
+|GA
+|DEP
+
 |====
 
 [id="ocp-4-5-deprecated-features"]
 === Deprecated features
 
+[id="ocp-4-5-deprecated-v1beta1-crds"]
+==== v1beta1 CRDs
+
+With the added support of v1 Custom Resource Definitions (CRDs) in Operator
```

Comment:
I'd suggest:

s/Custom Resource Definitions/CustomResourceDefinition/

It's a k8s resource. Or custom resource definition (CRD).